pkg/controllers/sync/controller.go

```diff
@@ -1294,3 +1300,12 @@ func convertSyncMapToMap(syncMap *sync.Map) map[interface{}]interface{} {
 
 	return normalMap
 }
+
+func skipSync(federatedObject fedcorev1a1.GenericFederatedObject) bool {
+	// if workloads have been propagated to member clusters,
```

Comment:
I think `no-sync` implies no sync in all scenarios, why do we differentiate here?